docs/cloud/provision/README.md

```diff
@@ -96,7 +96,11 @@ At this moment, it is not possible to change the cluster node instance size. You
 ![AWS cluster second part](./images/aws/aws-new-cluster-2.png)
 :::
 
-Further, you need to specify the storage capacity. One disk kind is available at the moment, but you can change the disk size. Since we allow customers to expand the storage size online without service interruptions, you can start with smaller storage and expand it when you need more capacity.
+Further, you need to specify the storage capacity. One disk kind is available at the moment, but you can change the disk size, IOPS and throughput. Since we allow customers to expand the storage size online without service interruptions, you can start with smaller storage and expand it when you need more capacity.
```

Comment:
I may be missing some context, but do we only present GP3 in the console and do these docs only apply there? The automation README above state `GP2` and `GP3` are supported.

Reply:
GP2 is still supported by API so customers will be able to use previous versions of terraform provider. Console allows to use only GP3.

Reply:
THat should be added as a note then .

Reply:
done